reference/high-availability-apps.html.md.erb

```diff
@@ -0,0 +1,121 @@
+
+This brings us to disaster recovery and being prepared. You should have backups of your data and understand how to recover that data if needed.
+
+## Fly.io high availability features
+
+The Fly Apps V2 platform has features to help make your app more resilient in case of outages or hardware failures.
+
+<div class="callout">
+<b>A note about Fly Proxy</b>: 
+Fly Proxy does a lot of work in the background, applying handlers, managing traffic and load balancing, and managing connections to services. Fly Proxy also automatically starts and stops Machines based on traffic to your app. The proxy works for [apps or processes with services defined](/docs/getting-started/app-services/#services-routed-with-fly-proxy).
+</div>
+
+The features for redundancy and availability are:
+
+* High availability by default on first deploy
+* Machines that start and stop automatically based on traffic
+* Health check-based routing
+
+These features require or depend on your app configuration in the fly.toml file. Learn more about [app configuration](/docs/reference/configuration/).
+
+You might also be interested in how we do [load balancing](/docs/reference/load-balancing/).
+
+## High availability by default on first deploy
+
+The first time you deploy your app to the Fly.io platform, you get some default high availability configurations. These settings help you to deploy a recommended setup without having to think about it. On the other hand, if the defaults aren't what you're looking for, then you can still configure your app and processes the way you want. Learn more about [scaling Machine CPU and RAM](/docs/apps/scale-machine/) and [scaling the number of Machines](/docs/apps/scale-count/).
+
+A Fly App with no extra process groups defined still uses the `app` process group by default. So there's no such thing as an app with no process groups.
+
+Here's what `fly launch` or `fly deploy` does, based on your app configuration:
+
+* creates and starts [two Machines](#two-machines-for-process-groups-with-services) for process groups with services, and sets automatic start and stop to true and minimum machines running to zero
+* creates and starts one Machine and creates one stopped [standby Machine](#standby-machines-for-process-groups-without-services) for process groups without services
+* creates and starts only one Machine if the process group has volumes mounted
+
+### Two Machines for process groups with services
+
+The most basic way to ensure high availability is to create more than one Machine per process group.
+
+If a process group has services defined, two Machines are automatically created and started when:
```

Comment:
It might be worth being explicit that this means people's apps will be load balanced by default, since that comes with a slew of concerns like "session storage" (to ensure people stay logged in when they're bounced between different servers), and "where do my files go?" - which is covered under the Volumes section, but might not be obvious that by default you need to care about that now.

This grouchy comment is brought to you by needing to adjust `fly launch` for Laravel apps because of session storage in load balanced environments 😂

Reply:
@fideloper Maybe this info belongs in "What your app needs to do"? Then I'd probably need to move the note about Fly Proxy up into that section too... 🤔